Resources/doc/index.rst

```diff
@@ -9,22 +9,24 @@ supports the `ORM`_ (MySQL, PostgreSQL, SQLite, etc.).
 Installation
 ------------
 
-Open a command console, enter your project directory and run the
-following command to download the latest stable version of this bundle:
+In Symfony 4 or higher applications that use `Symfony Flex`_, open a command
+console, enter your project directory and run the following command:
```

Comment:
This sentence sounds a bit weird to my ears… how about this instead :

Applications that use Symfony 4 or higher along with Symfony Flex`_, open a terminal…

Reply:
I've made some minor rewords. Thanks.